core/coreunix/add_test.go

```diff
@@ -54,7 +54,8 @@ func TestAddGCLive(t *testing.T) {
 
 	errs := make(chan error)
 	out := make(chan interface{})
-	adder, err := NewAdder(context.Background(), node, out)
+	adder, err := NewAdder(context.Background(), node.Pinning, node.Blockstore, node.DAG)
+	adder.Out = out
```

Comment:
just in case (even though its a test), i'd put the `adder.Out = out` after the error check. If this call does error for some reason, its going to return a nil adder and we will panic on this line and lose information about what the real error was